examples/example.qbs

```diff
@@ -0,0 +1,27 @@
+import qbs
+import "../box2d.qbs" as Box2D
+
+Project {
+    name: "Example";
+
+    Box2D { }
+
+    Product {
+        files: ["main.cpp"];
+        name: "Example";
+        type: "application";
```

Comment:
Hi,

Sorry for the delay

I'll answer on github

2015-07-17 20:51 GMT+02:00 Thorbjørn Lindeijer <notifications@github.com>:

> In examples/example.qbs
> <https://github.com/qml-box2d/qml-box2d/pull/102#discussion_r34919877>:
>
> > @@ -0,0 +1,27 @@
> > +import qbs
> > +import "../box2d.qbs" as Box2D
> > +
> > +Project {
> > +    name: "Example";
> > +
> > +    Box2D { }
> > +
> > +    Product {
> > +        files: ["main.cpp"];
> > +        name: "Example";
> > +        type: "application";
>
> Why not use the Application element instead?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/qml-box2d/qml-box2d/pull/102/files#r34919877>.
>